core/data/src/main/java/uk/gov/gchq/gaffer/data/generator/CsvFormat.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.data.generator;
+
+import java.util.LinkedHashMap;
+
+public interface CsvFormat {
+    LinkedHashMap<String, String> getIdentifiers();
```

Comment:
Shouldn't this be public? As it wouldn't be useful otherwise. This can be overridden to be wider, but it would be better if it's pubic here.